loan-payment-skip/LETTERSPECS/BANNO.LOANPAYMENT.SKIP.CFG

```diff
@@ -114,41 +123,147 @@ MM:0
 *      DEFAULT: 0
 *
 GD:
-
+*
 *Program time period - Which calendar dates is your skip program available?
 *     Line must begin with "TP:"
 *     data constraints - valid dates in the format MM/DD/YYYY-MM/DD/YYYY
 *      DEFAULT: (no defaults, valid start and end dates must be entered.
 *
-TP:10/01/2019-11/30/2019
-
+TP:
+*
 *Sub Source Code for Fee
 *     Line must begin with "SS:"
 *     data constraints: 0-50
 *      DEFAULT: 33 (Payment Skip Fee)
 *
 SS:33
-
+*
 *Other Action for Fee
 *     Line must begin with "OA:"
 *     data constraints: 0 or 1 -  Add comment to (0)share or (1)loan
 *      DEFAULT: 0
 *
 OA:
-
+*
 *Transaction comment for Fee
 *     Line must begin with "FC:"
 *     data constraints: Text comment
 *      DEFAULT: (left blank - no comment)*
 *
 FC:Skip Fee
-
+*
+** NEW PROGRAM OPTIONS 1/21/22
+*
+*Eligible Payment Frequencies
+* Only loans with one of the following payment frequencies will be eligible
+* for a skip payment
+*  Line must begin with "EPF:"
+*  DATA CONSTRAINTS: Comma delimited 2-digitnumeric list with values 01-09, 11-13

```

Comment:
Just looking at the configuration file for now.
The changes on the configuration file look good.

On line 161 I suggest changing "2-digitnumeric list" to
"2-digit numeric list"

Reply:
Corrected